src/components/ProfilePage/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect, useContext } from "react";
+import GamesCarousel from "../GamesCarousel";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+import { Col, Layout } from "antd";
+import profilePageImg from "../../images/profilePage/profilePageImg.png";
+import db from "../../firebaseConfig";
+import { v4 as uuidv4 } from "uuid";
+import { AuthContext } from "../../auth/authContext";
+
+const { Sider, Content } = Layout;
+const ProfilePage = () => {
+  const { t } = useTranslation();
+  const { currentUser } = useContext(AuthContext);
+  const currentUserId = currentUser ? currentUser.uid : null;
+  const [users, setUsers] = useState([]);
```

Comment:
why can there be multiple users for a single user id?

Reply:
i am not sure if i understood your question correctly  , so i said users because the collection in firebase called users which will be coming from the auth. is this what you ask about?